homeassistant/components/sensor/arest.py

```diff
@@ -91,7 +89,7 @@ def _render(value):
             renderer = make_renderer(var_data.get(CONF_VALUE_TEMPLATE))
             dev.append(ArestSensor(
                 arest, resource, config.get(CONF_NAME, response[CONF_NAME]),
-                variable, variable=variable,
+                var_data.get(CONF_NAME), variable=variable,
```

Comment:
Wouldn't it make more sense to make name optional and use the `variable` as a fallback for when `name` is not provided?

That way it is a backwards compatible change and also in sync with how we do it for groups etc.